Makefile

```diff
@@ -1,5 +1,13 @@
 lint:
+	poetry run black .
 	poetry run pyright .
```

Comment:
Same command is in lint-check. Shouldn't it be there only?

Reply:
My comment wasn't for black. It's for pyright.

Reply:
I think it's convenient to run a single command to see all the errors. During development you can just use `make lint` and automated tests use `make lint-check`

Reply:
I'm confused...
User also will be able to check errors with `make lint-check` if he wants to check lint.

And if he wants to fix it he will have `make lint`. 

Don't really understand why to mix showing errors and fixing errors.